scripts/bump.sh

```diff
@@ -23,21 +23,25 @@ NEXT_EXTENSION_VERSION=$3
 echo "NEXT_EXTENSION_VERSION: $NEXT_EXTENSION_VERSION"
 
 if [ "$CHANNEL" = "dev" ]; then
-    echo "$PRISMA_VERSION" > scripts/prisma_version_unstable
+    echo "$PRISMA_VERSION" > scripts/prisma_version_insider
 else
     echo "$PRISMA_VERSION" > scripts/prisma_version_stable
 fi
 
 # If the channel is dev, we need to change the name, displayName to the dev extension
 if [ "$CHANNEL" = "dev" ]; then
     jq ".version = \"$NEXT_EXTENSION_VERSION\" | \
-        .name = \"prisma-dev\" | \
-        .displayName = \"Prisma Dev\"" \
+        .name = \"prisma-insider\" | \
+        .displayName = \"Prisma - Insider\" | \
+        .description = \"This is the Insider Build.\" | \
```

Comment:
Is writing just this in the description intentional? I am probably missing some context of where this decision was made.

Reply:
I thought that this would make it more clear that this is not the extension users should use normally. Do you think we should add the original description here?

Reply:
I understood it as a good way to get less normal people to use this extension, which is a good idea. Maybe it could say:

```suggestion
        .description = \"This is the Insider Build of the Prisma VSCode extension.\" | \
```

Reply:
Once there is a different README for the insider version, we'll probably want to note that there too